### PR TITLE
Fixed merge conflict mishap

### PR DESCRIPTION
**Pull request by JoshArmi**

### CODE REVIEW

File: _posts/2018-08-06-better-bash-in-5-minutes.markdown [hunk 1]
```diff
@@ -1,11 +1,7 @@
 ---
 layout: post
 title:  "Better Bash Scripting in 5 Minutes or Less"
-<<<<<<< HEAD
-date:   2018-08-06
```
Comment (jgunnink) [inline _posts/2018-08-06-better-bash-in-5-minutes.markdown]: don't you want this date? Per your file name? Or is the filename wrong

Comment (Hamo69) [reply _posts/2018-08-06-better-bash-in-5-minutes.markdown]: date for Monday (next) please

Comment (Hamo69) [reply _posts/2018-08-06-better-bash-in-5-minutes.markdown]: which it is, right?

Comment (Hamo69) [reply _posts/2018-08-06-better-bash-in-5-minutes.markdown]: what

Comment (Hamo69) [reply _posts/2018-08-06-better-bash-in-5-minutes.markdown]: happens

Comment (Hamo69) [reply _posts/2018-08-06-better-bash-in-5-minutes.markdown]: if

Comment (Hamo69) [reply _posts/2018-08-06-better-bash-in-5-minutes.markdown]: I

**Pull request merged**